core/migrations/0008_auto_20211130_1437.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0007_index_issue_titles_with_dates'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='issuenote',
+            name='text',
+            field=models.TextField(default=b'', blank=True),
```

Comment:
Does this not work with a Unicode string? (Recognizing that this app really needs to be upgraded to Python 3)

Reply:
I am not sure why Django made this into bytes. As you can see in the `models.py` file, I just made it a blank string.

Reply:
Hmm, that's definitely curious but I suspect the answer in all cases will be upgrading.